frontend/src/scenes/billing/BillingHero.tsx

```diff
@@ -1,26 +1,220 @@
+                <p className="italic">P.S. You still keep the monthly free allotment for every product!</p>
+            </>
+        ),
+    },
+    [BillingPlan.Paid]: {
+        title: 'Good call!',
+        subtitle: "You're on the Ridiculously Cheap™ plan.",
+        backgroundColor: 'bg-warning-highlight',
+        getDescription: (_billingPlan: BillingPlan, scrollToProduct: (productType: string) => void) => (
+            <p>
+                If you're growing like crazy, you might want to check out the{' '}
+                {scrollToProduct ? (
+                    <>
+                        <Link onClick={() => scrollToProduct('teams')}>Teams</Link>
+                        {' or '}
+                        <Link onClick={() => scrollToProduct('enterprise')}>Enterprise</Link>
+                    </>
+                ) : (
+                    'Teams or Enterprise'
+                )}{' '}
+                plan.
+            </p>
+        ),
+    },
+    [BillingPlan.Teams]: {
+        title: 'Good call!',
+        subtitle: "You're on the Teams plan.",
+        backgroundColor: 'bg-warning-highlight',
+        getDescription: (_billingPlan: BillingPlan, scrollToProduct: (productType: string) => void) => (
+            <p>
+                If you're growing like crazy, you might want to check out the{' '}
+                {scrollToProduct ? <Link onClick={() => scrollToProduct('enterprise')}>Enterprise</Link> : 'Enterprise'}{' '}
+                plan.
+            </p>
+        ),
+    },
+    [BillingPlan.Enterprise]: {
+        title: 'Good call!',
+        subtitle: "You're on the Enterprise plan.",
+        backgroundColor: 'bg-success-highlight',
+        getDescription: () => <p>It doesn't get any better than this!</p>,
+    },
+    [StartupProgramLabel.Startup]: {
+        title: 'Good for you!',
+        subtitle: "You're on the startup plan.",
+        backgroundColor: 'bg-warning-highlight',
+        getDescription: (billingPlan: BillingPlan, scrollToProduct: (productType: string) => void) => (
+            <p>
+                If you're growing like crazy, you might want to check out the{' '}
+                {billingPlan !== BillingPlan.Teams ? (
+                    <>
+                        {scrollToProduct ? (
+                            <>
+                                <Link onClick={() => scrollToProduct('teams')}>Teams</Link>
+                                {' or '}
+                            </>
+                        ) : (
+                            'Teams or '
+                        )}
+                    </>
+                ) : null}
+                {scrollToProduct ? <Link onClick={() => scrollToProduct('enterprise')}>Enterprise</Link> : 'Enterprise'}{' '}
+                plan.
+            </p>
+        ),
+    },
+    [StartupProgramLabel.YC]: {
+        title: 'Lucky you!',
+        subtitle: "You're on the special YC plan.",
```

Comment:
```suggestion
        subtitle: "You're on the YC plan.",
```

Cutting this just removes the overrun on the line